src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Data/Permission.cs

```diff
@@ -6,6 +6,7 @@ public class Permission
 {
     public int Id { get; set; }
     public string UserId { get; set; } = string.Empty;
+    public string Email { get; set; } = string.Empty;
```

Comment:
Do we still need this?  This one has potential to get stale and out of sync with IDP records.

Reply:
Ah, thanks for catching that. I meant to remove. Should be gone now